rules/builds.smk

```diff
@@ -298,15 +294,15 @@ rule adjust_metadata_regions:
     output:
         metadata = "results/{build_name}/metadata_adjusted.tsv"
     params:
-        geographic_scale = lambda wildcards: _get_geographic_scale_by_build_name(wildcards.build_name),
+        subsampling_scheme = lambda wildcards: _get_subsampling_scheme_by_build_name(wildcards.build_name),
         geographic_name = lambda wildcards: _get_geographic_name_by_build_name(wildcards.build_name)
     log:
         "logs/adjust_metadata_regions_{build_name}.txt"
     conda: config["conda_environment"]
     shell:
         """
         python3 scripts/adjust_regional_meta.py \
-            --{params.geographic_scale} {params.geographic_name:q} \
+            --{params.subsampling_scheme} {params.geographic_name:q} \
```

Comment:
This needs to be geographic scale here, I think. Otherwise the adjust metadata script won't know what to do with subsampling schemes like "switzerland".

Reply:
you are right!